### PR TITLE
Support Carthage

### PR DESCRIPTION
**Pull request by luosheng**
Create two shared schemes for iOS and OS X respectively.

### CODE REVIEW

**Comment by krzyzanowskim** [comment]
please apply this to branch "swift12" as well.

**Comment by luosheng** [comment]
I thought this pull request was applied to branch "swift12". Did you mean "master"?

**Comment by krzyzanowskim** [comment]
ah ;) sorry. Yea, give it to master as well.

**Comment by krzyzanowskim** [comment]
but wait
did you change name of the framework to CryptoSwift-iOS etc? This would impact current instalation, cocoapods especially right ?

**Comment by luosheng** [comment]
I don't think it would impact CocoaPods but let me give it a try first.

**Comment by luosheng** [comment]
So I did a little experiment and my project compiled and ran well. Here're the Podfile I wrote and the Podflie.lock generated. I think it's safe to say that the change won't impact CocoaPods installation.

Podfile:

```ruby
use_frameworks!

pod 'CryptoSwift', :git => "https://github.com/luosheng/CryptoSwift", :branch => "swift12"
```

Podfile.lock:
```
PODS:
  - CryptoSwift (0.0.7)

DEPENDENCIES:
  - CryptoSwift (from `https://github.com/luosheng/CryptoSwift`, branch `swift12`)

EXTERNAL SOURCES:
  CryptoSwift:
    :branch: swift12
    :git: https://github.com/luosheng/CryptoSwift

CHECKOUT OPTIONS:
  CryptoSwift:
    :commit: 41a583c4b521ac484bc1c2209a08cabd7ded8a27
    :git: https://github.com/luosheng/CryptoSwift

SPEC CHECKSUMS:
  CryptoSwift: 9ac21389daa3de6395c527e51715e908b993f3ed

COCOAPODS: 0.36.0
```

Regarding making the same change on `master`, maybe we can merge this PR to `swift12` and cherry-pick this commit to `master`?

File: CryptoSwift.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -666,6 +941,104 @@
...
+					"$(SDKROOT)/Developer/Library/Frameworks",
+					"$(inherited)",
+				);
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"DEBUG=1",
+					"$(inherited)",
+				);
+				HEADER_SEARCH_PATHS = (
+					"$(inherited)",
+					"/Applications/Xcode6-Beta3.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include",
```
Comment (krzyzanowskim) [inline CryptoSwift.xcodeproj/project.pbxproj]: ?

**Comment by krzyzanowskim** [comment]
I found hardcoded path: "/Applications/Xcode6-Beta3.app" this shouldn't be part of SCM :) And I don't want rename target to CryptoSwift-iOS, why this necessary for Carthage?

**Pull request closed without merging**